src/optimalcontrol/CMakeLists.txt

```diff
@@ -7,8 +7,6 @@ set(libraryname idyntree-optimalcontrol)
 
```

Comment:
I cannot comment above this point, but in all CMakeLists.txt you can safely use CMake 3.5.

Reply:
I was keeping consistency with the base ``CMakeLists.txt``.